ghost/ghost/src/core/activitypub/actor.entity.ts

```diff
@@ -16,7 +16,9 @@ type ActorData = {
     outbox: Activity[];
     inbox: Activity[];
     following: {id: URI, username?: string;}[];
-    followers: {id: URI;}[];
+    followers: {id: URI;}[],
```

Comment:
@allouis I wasn't sure if we need `followers` here. There's some funky logic with `activities` that I haven't fully gotten yet, so maybe it's wrong to add it here?